tests/API/AuthenticationTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace vendor\statamic\cms\tests\API;
+
+use PHPUnit\Framework\Attributes\Test;
+use Statamic\Facades;
+use Tests\PreventSavingStacheItemsToDisk;
+use Tests\TestCase;
+
+class AuthenticationTest extends TestCase
+{
+    use PreventSavingStacheItemsToDisk;
+
+    private $collection;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        Facades\Config::set('statamic.api.enabled', true);
+        Facades\Config::set('statamic.api.resources.collections', true);
+
+        $this->collection = Facades\Collection::make('articles')->save();
+    }
+
+    #[Test]
+    public function it_can_authenticate_using_api_token()
+    {
+        Facades\Config::set('statamic.api.api_token', 'foobar');
+
+        $this
+            ->getJson('/api/collections/articles/entries', [
```

Comment:
why not use `public function withToken(string $token, string $type = 'Bearer')` here?

Reply:
Because I didn't know that method existed 😄